tsfile/src/test/java/org/apache/iotdb/tsfile/write/MetadataIndexConstructorTest.java

```diff
@@ -49,11 +49,7 @@
 
 import java.io.File;
 import java.io.IOException;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
```

Comment:
Please use single class import instead of wildcard

Reply:
Sorry I'm trying to fix the code format in my new laptop today. This is fixed in the latest commit : )